com.unity.render-pipelines.high-definition/Runtime/Lighting/Light/HDAdditionalLightData.cs

```diff
@@ -2855,20 +2855,17 @@ internal void UpdateAreaLightEmissiveMesh(bool fromTimeLine = false)
             }
 
             // Update Mesh
-            if (HDRenderPipeline.defaultAsset != null)
+            switch (areaLightShape)
             {
-                switch (areaLightShape)
-                {
-                    case AreaLightShape.Tube:
-                        if (m_EmissiveMeshFilter.sharedMesh != HDRenderPipeline.defaultAsset.renderPipelineResources.assets.emissiveCylinderMesh)
-                            m_EmissiveMeshFilter.sharedMesh = HDRenderPipeline.defaultAsset.renderPipelineResources.assets.emissiveCylinderMesh;
-                        break;
-                    case AreaLightShape.Rectangle:
-                    default:
-                        if (m_EmissiveMeshFilter.sharedMesh != HDRenderPipeline.defaultAsset.renderPipelineResources.assets.emissiveQuadMesh)
-                            m_EmissiveMeshFilter.sharedMesh = HDRenderPipeline.defaultAsset.renderPipelineResources.assets.emissiveQuadMesh;
-                        break;
-                }
+                case AreaLightShape.Tube:
+                    if (m_EmissiveMeshFilter.sharedMesh != HDRenderPipelineGlobalSettings.instance.renderPipelineResources.assets.emissiveCylinderMesh)
+                        m_EmissiveMeshFilter.sharedMesh = HDRenderPipelineGlobalSettings.instance.renderPipelineResources.assets.emissiveCylinderMesh;
+                    break;
+                case AreaLightShape.Rectangle:
+                default:
+                    if (m_EmissiveMeshFilter.sharedMesh != HDRenderPipelineGlobalSettings.instance.renderPipelineResources.assets.emissiveQuadMesh)
+                        m_EmissiveMeshFilter.sharedMesh = HDRenderPipelineGlobalSettings.instance.renderPipelineResources.assets.emissiveQuadMesh;
+                    break;
```

Comment:
Before we checked against null asset. It is not done anymore. The HDRenderPipelineGlobalSettings.instance can still be null?

Reply:
(the null check was just above. I forgot to take it in the scope)

Reply:
i checked the code and i removed the null check on purpose because i dont see any valid reason  to not have that code be run. as long as you have HDRP in your project, active or not, you could always gather info from the global settings thus display an HDRP Additional Data Light
Is my assumption wrong?

Reply:
Can user destroy the global settings asset ?

Reply:
ok i will fix the destroy workflow asset and re-add the necessary checks